sdk/core/Azure.Core/src/RetryOptions.cs

```diff
@@ -11,17 +11,27 @@ namespace Azure.Core
     /// </summary>
     public class RetryOptions
     {
-        internal RetryOptions()
+        /// <summary>
+        /// Creates a new <see cref="RetryOptions"/> instance with default values.
+        /// </summary>
+        protected internal RetryOptions() : this(ClientOptions.Default.Retry)
```

Comment:
Is this being used currently? Or are we just enabling it now because we plan to use it in the future and don't want to add another overload ctor to ClientOptions?

Reply:
Correct - see [this comment](https://github.com/Azure/azure-sdk-for-net/pull/27520/files#r835436473) for a bit more detail.

Reply:
make internal